packages/taquito-contract-type-generator/example/example-usage.ts

```diff
@@ -0,0 +1,207 @@
+import { TezosToolkit } from '@taquito/taquito';
+import { Contract as TestContractType } from './types/example-contract-1';
+import { Contract as TestContractType2 } from './types/example-contract-2';
+import { nat, tas } from './types/type-aliases';
+import { createContractAbstractionComposer, TezosToolkitTyped, TypedContractAbstractionOf } from '../src/usage/tezos-toolkit-typing';
+
+
+const exampleContractMethods1 = async () => {
+
+    // TODO: Support Generic Type in TezosToolkit Directly
+    // const Tezos = new TezosToolkit<TestContractType>(`https://YOUR_PREFERRED_RPC_URL`)
+    const Tezos = new TezosToolkit(`https://YOUR_PREFERRED_RPC_URL`) as unknown as TezosToolkitTyped<TestContractType>;
+
+    const contract = await Tezos.contract.at(`tz123`);
+
+    contract.methods.bid(tas.nat(0));
+    contract.methods.configure({
```

Comment:
I think this example is not right as the contract methods do not expect an object but flat parameters.  We have an internal discussion about accepting an object for the methods (like we do for the storage) as it is easier to reason about, but this would be a breaking change...
But that said, the work you did in this package makes it easier to discover what parameters are expected, this is really helpful.

Reply:
Right, this is wrong - leftover from how I first did it.

Thanks!